.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+_[One line description of your change]_
+
+### Motivation:
+
+_[Explain here the context, and why you're making that change. What is the problem you're trying to solve.]_
```

Comment:
nitpick: My personal pet peeve about those is "link back to issue which this PR is addressing"
// I know the template is taken from NIO which does not include the hint, though could be a good one to add?

Reply:
how about we take this in, then do a separate PR in NIO and then cascade to all others?

Reply:
Sounds good 👍